docs/security/host-certs/overview.md

```diff
@@ -33,23 +55,10 @@ We recommend requesting host certificates from one of the following CAs:
 If neither of the above options work for your site, the OSG also accepts all
 [IGTF-accredited CAs](https://repo.opensciencegrid.org/cadist/).
 
-Before Starting
----------------
-
-Before requesting a new host certificate, use `openssl` to check if your host already has a valid certificate, i.e. the
-present is between `notBefore` and `notAfter` dates and times.
-If so, you may safely skip this document:
-
-``` console
-user@host $ openssl x509 -in /etc/grid-security/hostcert.pem -subject -issuer -dates -noout
-subject= /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=host.opensciencegrid.org
-issuer=/DC=org/DC=cilogon/C=US/O=CILogon/CN=CILogon OSG CA 1
-notBefore=Jan  4 21:08:09 2010 GMT
-notAfter=Jan  4 21:08:09 2011 GMT
-```
-
-If you are using OpenSSL 1.1, you may notice minor formatting differences.
 
+!!!note
+    For SSL to work properly, you will need to request a host certificate with
+    "TLS Web Server Authentication" included in the X509v3 Extended Key Usage.
```

Comment:
```suggestion
    "TLS Web Server Authentication" and "TLS Web Client Authentication" included
    in the X509v3 Extended Key Usage.
```

Come to think of it (I know it's not noted in the original ticket), users probably also need "TLS Web Client Authentication"